.semaphore/semaphore.yml

```diff
@@ -4,6 +4,10 @@ agent:
   machine:
     type: s1-prod-ubuntu20-04-amd64-1
 
+# Increase time limit to allow tests to run to completion
+execution_time_limit:
+  hours: 2
```

Comment:
I'm surprised it takes that long now, we used to be able to run tests much faster than that in the past, but we can leave that until we figure out what it taking so long

Reply:
A similar setting on another repo has a comment: `# Increase time limit to occasionally allow for cache population.` 
Which could have been the case here as well, given that the successful run took 40min. Reverting this patch and we can reconsider if this starts happening more frequently.